sn_client/src/api.rs

```diff
@@ -144,13 +151,13 @@ impl Client {
                 }
                 Ok(ClientEvent::InactiveClient(timeout)) => {
                     let random_target = ChunkAddress::new(XorName::random(&mut rng));
-                    debug!("No ClientEvent activity in the past {timeout:?}, performing a random get_chunk query to target: {random_target:?}");
+                    debug!("No ClientEvent activity in the past {timeout:?}, performing a random get_store_cost query to target: {random_target:?}");
                     if is_connected {
-                        println!("The client experienced inactivity in the past {timeout:?}.");
+                        info!("The client was inactive for {timeout:?}.");
                     } else {
-                        println!("The client still does not know enough network nodes.");
+                        info!("The client still does not know enough network nodes.");
                     }
-                    let _ = client.get_chunk(random_target).await;
+
```

Comment:
I think a call to fetch a random `get_store_cost` is missing here.

Reply:
I removed that intentionally as it doesn't seem to help much at all, and confuses the logs